components/Decks.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { View, Text, FlatList, StyleSheet } from "react-native";
+
+const Decks = () => {
+  const testDecks = [
```

Comment:
This is mock data to be removed as soon as backend is deployed correct ?

Reply:
Yes correct, we will render the state currentDecks when we can call to the BE